pkg/util/cgroup/manager/v2/fs_linux.go

```diff
@@ -219,6 +219,34 @@ func (m *manager) ApplyIOWeight(absCgroupPath string, devID string, weight uint6
 	return nil
 }
 
+func (m *manager) ApplyIOLatency(absCgroupPath string, devID string, latency uint64) error {
+	var dataContent string
+	if latency == 0 {
+		dataContent = fmt.Sprintf("%s target=max", devID)
+	} else {
+		dataContent = fmt.Sprintf("%s target=%d", devID, latency)
+	}
+
+	curLatency, found, err := m.GetDeviceIOLatency(absCgroupPath, devID)
```

Comment:
pls remove those unnecessary blanks (not only for this specific one but for all newly added codes)